nx.json

```diff
@@ -152,6 +152,17 @@
         "runtime": "node --version"
       }
     ],
+    "DevOps": ["CI", "Dockerfiles"],
+    "CI": [
+      "{workspaceRoot}/**/docker-compose*.yml",
```

Comment:
Which docker files are these? Can we make this more focused? 

We have plenty of docker-compose files in some apps which are only (?) used for development. These are already matched by `production/default` with `"{projectRoot}/**/*"`. We don't want application system docker-compose to affect all applications.

Maybe you want to mainly target stuff in the infra or script folders?

Reply:
Removed docker-compose, makes sense not to include development docker-compose.

I could list all the Dockerfiles I know of, and that we have currently, but no app should use Dockerfiles, and any Dockerfile created should be part of our bulid/CI. I'd like to opt for a simpler wider include rather than specifying the Dockerfiles we need; since currently, it's _all_ the Dockerfiles.